dll/win32/shell32/propsheet.cpp

```diff
@@ -160,6 +160,44 @@ SHELL32_OpenPropSheet(LPCWSTR pszCaption, HKEY *ahKeys, UINT cKeys,
     return (Result != -1);
 }
 
+/*************************************************************************
+ *  SHOpenPropSheetA [SHELL32.707]
+ *
+ * @see https://learn.microsoft.com/en-us/windows/win32/api/shlobj/nf-shlobj-shopenpropsheeta
+ */
+EXTERN_C
+BOOL WINAPI
+SHOpenPropSheetA(
+    _In_opt_ LPCSTR pszCaption,
+    _In_opt_ HKEY *ahKeys,
+    _In_ UINT cKeys,
+    _In_ const CLSID *pclsidDefault,
+    _In_ IDataObject *pDataObject,
+    _In_opt_ IShellBrowser *pShellBrowser,
+    _In_opt_ LPCSTR pszStartPage)
+{
+    CStringW strStartPageW, strCaptionW;
+    LPCWSTR pszCaptionW = NULL, pszStartPageW = NULL;
+
+    TRACE("(%s, %p, %u, %p, %p, %p, %s)", debugstr_a(pszCaption), ahKeys, cKeys, pclsidDefault,
```

Comment:
It's doubtful we need the trace. This is a habit of Wine to systematically add traces at the beginning of functions, which isn't much useful (one can use a debugger or enable relay-tracing to view that information).

Reply:
Deleted.